scenarios/kubernetes_e2e.py

```diff
@@ -0,0 +1,195 @@
+    gce_pub = '%s.pub' % gce_ssh
+    service = '/service-account.json'
+
+    cmd.extend([
+      '-v', '%s:%s:ro' % (args.gce_ssh, gce_ssh),
+      '-v', '%s:%s:ro' % (args.gce_pub, gce_pub),
+      '-v', '%s:%s:ro' % (args.service_account, service),
+      '-e', 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE=%s' % gce_ssh,
+      '-e', 'JENKINS_GCE_SSH_PUBLIC_KEY_FILE=%s' % gce_pub,
+      '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service,
+      # Boilerplate envs
+      # Skip gcloud update checking
+      '-e', 'CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=\'true\'',
+      # Use default component update behavior
+      '-e', 'CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE=\'false\'',
+      # E2E
+      '-e', 'E2E_UP=%s' % args.e2e_up,
+      '-e', 'E2E_TEST=%s' % args.e2e_test,
+      '-e', 'E2E_DOWN=%s' % args.e2e_down,
+      '-e', 'E2E_NAME=%s' % args.e2e_name,
+      # AWS
+      '-e', 'KUBE_AWS_INSTANCE_PREFIX=%s' % args.e2e_name,
+      # GCE
+      '-e', 'INSTANCE_PREFIX=%s' % args.e2e_name,
+      '-e', 'KUBE_GCE_NETWORK=%s' % args.e2e_name,
+      '-e', 'KUBE_GCE_INSTANCE_PREFIX=%s' % args.e2e_name,
+      # GKE
+      '-e', 'CLUSTER_NAME=%s' % args.e2e_name,
+      '-e', 'KUBE_GKE_NETWORK=%s' % args.e2e_name,
+      # Workspace
+      '-e', 'HOME=/workspace',
+      '-e', 'WORKSPACE=/workspace'])
+
+    # env blacklist.
+    # TODO(krzyzacy) change this to a whitelist
+    docker_env_ignore = [
+      'GOOGLE_APPLICATION_CREDENTIALS',
+      'GOROOT',
+      'HOME',
+      'PATH',
+      'PWD',
+      'WORKSPACE'
+    ]
+
+    for key, value in os.environ.items():
+        if key not in docker_env_ignore:
+            cmd.extend(['-e', '%s=%s' % (key, value)])
+
+    cmd.extend([
+      'gcr.io/k8s-testimages/kubekins-e2e:%s' % e2e_image_tag
+    ])
+
+    signal.signal(signal.SIGTERM, sig_handler)
+    signal.signal(signal.SIGINT, sig_handler)
+
+    try:
+        check(*cmd)
+        print 'Finished Successfully'
+    except subprocess.CalledProcessError as exc:
+        print >>sys.stderr, 'Exiting with code: %d' % exc.returncode
+        raise exc
+
+
+if __name__ == '__main__':
+
+    PARSER = argparse.ArgumentParser()
+    PARSER.add_argument(
+        '--env-file', action="append", help='Job specific environment file')
+
+    PARSER.add_argument(
+        '--gce-ssh',
+        default=os.environ.get('JENKINS_GCE_SSH_PRIVATE_KEY_FILE'),
+        help='Path to .ssh/google_compute_engine keys')
+
+    PARSER.add_argument(
+        '--gce-pub',
+        default=os.environ.get('JENKINS_GCE_SSH_PUBLIC_KEY_FILE'),
+        help='Path to pub gce ssh key')
+
+    PARSER.add_argument(
+        '--service-account',
+        default=os.environ.get('GOOGLE_APPLICATION_CREDENTIALS'),
+        help='Path to service-account.json')
+
+    # Assume we're upping, testing, and downing a cluster by default
+    PARSER.add_argument(
+        '--e2e-up', default='true', help='If we need to set --up in e2e.go')
```

Comment:
Given that the scenario is kubernetes_e2e.py it seems redundant to prefix flags with --e2e, especially since not all the flags have this prefix.